torch/csrc/distributed/c10d/intra_node_comm.cpp

```diff
@@ -299,6 +299,10 @@ c10::intrusive_ptr<IntraNodeComm> IntraNodeComm::rendezvous(
     return nullptr;
   }
 
+  if (bufferSize == 0) {
```

Comment:
🤔 what does this mean when passing in a `buffSize == 0`? I think the user intention on this might be they want to disable intra node comm?

Reply:
Yeah this is confusing. 0 here means "unspecified and use default value". Let me change it to `c10::optional` or something.